### DISCUSSION TITLE
Audio Gridder VST3 plugin not showing up in Ableton Live on M1

### DISCUSSION THREAD

**Discussion by prakir77** [apohl79/audiogridder#1139] [Help]
Could somebody help out with this issue?

Loading Audio Gridder as a VST3 plugin works fine when Ableton Live is opened in Rosetta 2. However, when I switch to running Ableton Live as a native ARM application, Audio Gridder is nowhere to be found among the list of plugins. As a result, I am unable to load Audio Gridder as a VST3 plugin when running daw in Apple silicon mode. Projects containing the Audio Gridder VST3 plugin load fine in Rosetta 2, but display “missing plugin could not be found” message in ARM. Running latest Ableton Live 11 on macOS Ventura and latest Audio Gridder 1.2.0 beta release 15.

Thanks.

**Comment by apohl79** [comment]
Please install the universal build of the plugin.

**Reply by prakir77** [reply]
Thank you for your reply, the Universal build is what I had installed on my system. I also made sure that "Run in Rosetta" wasn't enabled on any of the Audio Grider applications (as well as the host DAW Ableton), but nothing seemed to work. What did work was I actually had to uninstall the Universal build and install the ARM only version. It seems to have resolved the issue, as everything appears to be working now. Thank you very much for the reply though, appreciate it!
